E2E-ORS-Sample-For-Java/src/main/java/objectReplicationMonitor.java

```diff
@@ -0,0 +1,329 @@
+
+            // get the container client to access blob client
+            BlobContainerClient blobContainerClient = blobServiceClient.getBlobContainerClient(containerName[s]);
+
+            // print out contents of one blob to show successful replication
+            BlobClient blobClient = blobContainerClient.getBlobClient((String) blobList.get(0));
+
+            System.out.println("\nContent of " + blobList.get(0) + " in the " + containerName[s] + " container:");
+
+            // create local file to store contents
+            File localFile = new File("src/main/java/replicated_blob_content.txt");
+
+            // download blob content to file
+            blobClient.downloadToFile("src/main/java/replicated_blob_content.txt", true);
+
+            // read content of file
+            Scanner fileReader = new Scanner(localFile);
+            while (fileReader.hasNextLine()) {
+
+                String content = fileReader.nextLine();
+                System.out.println(content);
+
+            }
+
+            fileReader.close();
+
+            // delete content file-- clean up
+            localFile.deleteOnExit();
+        }
+
+    }
+
+
+    private static void archiveReplicatedDataWithBatch(String destinationConnectionString, String destinationContainer, ArrayList blobList) {
+        /*
+        this method is to archive the replicated blobs with batch
+         */
+
+        System.out.println("\nTransferring replicated blobs in destination container to archive tier...");
+        System.out.println("This may take a few minutes...");
+
+        // get blobServiceClient to create batch client
+        BlobServiceClient blobServiceClient = new BlobServiceClientBuilder().connectionString(destinationConnectionString).buildClient();
+
+        // create a batch client that can be used to archive multiple blobs
+        BlobBatchClient blobBatchClient = new BlobBatchClientBuilder(blobServiceClient).buildClient();
+
+        // get all blobs in container's urls and add them to a list
+        List<String> blobUrls = new ArrayList<>();
+
+        for (Object b : blobList) {
+
+            String url = blobServiceClient.getBlobContainerClient(destinationContainer).getBlobClient((String) b).getBlobUrl();
+            blobUrls.add(url);
+
+        }
+
+        // set the access tier of the batch to Archive with try/catch
+        try {
+
+            blobBatchClient.setBlobsAccessTier(blobUrls, AccessTier.ARCHIVE).forEach(response ->
+                    System.out.printf("Setting blob access tier with URL %s completed with status code %d%n",
+                            response.getRequest().getUrl(), response.getStatusCode()));
+
+            System.out.println("\nArchived replicated blobs!");
+
+        }
+
+        catch (BlobBatchStorageException ex) {
+
+            for (BlobStorageException e : ex.getBatchExceptions()) {
+
```

Comment:
I think we can remove this error handling stuff, or remove the break so you can print all the errors

Reply:
I think i'll remove the break rather than removing it all, but i might included a print statement with something along the lines of "make sure you used http and disabled secure transfer" to reiterate what is required to use batch